apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -466,7 +471,7 @@ export const CssValueInput = ({
             }}
             onBlur={handleOnBlur}
             onKeyDown={handleKeyDown}
-            baseRef={scrubRef}
+            containerRef={scrubRef}
```

Comment:
can you please make it first throw error from typescript?

Reply:
The issue with getBlablaProps in downshift that each getter can have input which is going into out.
And they decided to use `any` type for getInputProps() Return type. Removal of spread in case of downshift could cause more serious issues and harder to find bugs vs we already had. 
That needs to reimplement all downshift types first. 
IMO can be done but not in this PR